build/build-config-edit.sh

```diff
@@ -22,7 +23,7 @@ do
     else
         exe_extension=""
     fi
-    outputFile="${bindir}/config-edit-${go_os}${exe_extension}"
+    outputFile="${bindir}/${go_os}/config-edit-${go_os}${exe_extension}"
```

Comment:
It makes the bin folder more neat. Right now all binaries are in the same directory.